internal/app/server/halder_test.go

```diff
@@ -1 +1,91 @@
 package server
+
+import (
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"parcel-service/internal/app/model"
+	"parcel-service/internal/app/service/mocks"
+	"strings"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
```

Comment:
don't use const at here. its only used by one test case so put it inside that test case